test/unit/parser_test.rb

```diff
@@ -74,4 +74,28 @@ def test_command_type_returns_arithmetic_command
     parser.advance
     assert_equal(:C_ARITHMETIC, parser.command_type)
   end
+
```

Comment:
There are assertions in several earlier tests (e.g. `# assert_equal("push constant 17", parser.comp)`) which should probably be uncommented as part of this change.